modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/DXPActivationKeysTable/index.js

```diff
@@ -0,0 +1,465 @@
+								activationKeys?.statusBar
+									?.notActiveTotalCount || 0
+							})`,
+							value: ACTIVATION_STATUS.notActivated.id,
+						},
+						{
+							label: `${ACTIVATION_STATUS.expired.title} (${
+								activationKeys?.statusBar?.expiredTotalCount ||
+								0
+							})`,
+							value: ACTIVATION_STATUS.expired.id,
+						},
+					]}
+					onChange={(value) => {
+						setFilterStatusBar(value);
+					}}
+				/>
+			</div>
+
+			<ClayTooltipProvider
+				contentRenderer={(props) => {
+					const activationNames = props.title.split(',');
+
+					if (activationNames.length) {
+						return (
+							<div>
+								<p className="font-weight-bold m-0">
+									{activationNames[0]}
+								</p>
+
+								<p className="font-weight-normal m-0 text-paragraph-sm">
+									{activationNames[1]}
+								</p>
+							</div>
+						);
+					}
+
+					return activationNames[0];
+				}}
+				delay={100}
+			>
+				<Table
+					className="border-0 cp-dxp-activation-key-table mt-5"
+					columns={columns}
+					hasCheckbox
+					hasPagination
+					isLoading={isLoadingActivationKeys}
+					paginationConfig={{
+						activePage,
+						itemsPerPage,
+						labels: {
+							paginationResults: 'Showing {0} to {1} of {2}',
+							perPageItems: 'Show {0} Items',
+							selectPerPageItems: '{0} Items',
+						},
+						listItemsPerPage: [
+							{label: 5},
+							{label: 10},
+							{label: 20},
+							{label: 50},
+						],
+						setActivePage,
+						setItemsPerPage,
+						showDeltasDropDown: true,
+						totalCount,
+					}}
+					rows={activationKeysFiltered.map((activationKey) => ({
+						download: (
+							<IconButton
+								displayType="null"
+								onClick={() =>
+									handleDownloadActivationLicenseKey(
+										activationKey.id
+									)
+								}
+								small
+								symbol="download"
+							/>
+						),
+						envName: (
+							<div
+								title={[
+									activationKey.name,
+									activationKey.description,
+								]}
+							>
+								<p className="font-weight-bold m-0 text-neutral-10 text-truncate">
+									{activationKey.name}
+								</p>
+
+								<p className="font-weight-normal m-0 text-neutral-7 text-paragraph-sm text-truncate">
+									{activationKey.description}
+								</p>
+							</div>
+						),
+						envType: (() => {
+							const productName = getPascalCase(
+								activationKey.licenseEntryType
+							)
+								.split('-')
+								.join(' ');
+
+							const productDescription = activationKey.complimentary
+								? 'Complimentary'
+								: 'Subscription';
+
+							return (
+								<div>
+									<p className="font-weight-bold m-0 text-neutral-10">
+										{productName.replace(
+											'Production',
+											'Prod'
+										)}
+									</p>
+
+									<p className="font-weight-normal m-0 text-neutral-7 text-paragraph-sm">
+										{productDescription}
+									</p>
+								</div>
+							);
+						})(),
+						expirationDate: (() => {
```

Comment:
Should be good not use function, try to mount it before and just use it as component values